src/docs/product/issues/issue-details/performance-issues/consecutive-http/index.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: "Consecutive HTTP"
+sidebar_order: 30
+redirect_from:
+  - /product/issues/performance-issues/consecutive-http/
+description: "Learn more about Consecutive HTTP and how to diagnose and fix them."
+---
+
+<Include name="early-adopter-note.mdx" />
+
+Consecutive HTTP issues are created when a set of sequential http spans are consistently exceeding a duration threshold.
+
+## Detection Criteria
+
+The detector for this performance issue looks for a set of sequential, non-overlapping http spans. It intentionally ignores non-http spans when finding this set, as the browser commonly performs other asynchronous operations such as loading resources, between http spans. 
+Once this sequence is found, the following must hold true for each HTTP span.
+
+- The http method must be a GET, POST, DELETE, PUT, PATCH
+- The http url must not begin with _next/static/ or _next/data/
+
+Once these spans are found, the following must also hold true:
+
+- The duration of each http span must exceed a threshold.
+- The time between each http span must exceed a threshold.
+- The number of sequential http spans must exceed a threshold.
+
+If Sentry isn't detecting a Consecutive http issue where you expect one, it's probably because the transaction didn't meet one of the above criteria.
+
+## Span Evidence
+
+You can identify a Consecutive http problem by four main aspects in the "Span Evidence" section:
+
+- Transaction name
+- Sequential HTTP Spans
+
+![Consecutive HTTP span evidence](span-evidence.png)
+
+View it by going to the **Issues** page in Sentry, selecting your project, clicking on the Consecutive HTTP Queries error you want to examine, then scrolling down to the "Span Evidence" section in the "Details" tab.
+
+## Fixing Consecutive HTTP Issues
+
+The reasons for a consecutive http issue varies, but here are some suggestions to help you resolve them.
+- If each http span is independent of one another, consider parallelzing your http spans so they are no longer sequential.
+- If you have control over the backend, consider to optimize your endpoints for this specific use case. You can dive into the corresponding backend transactions to help identify potential sources of improvement.
+- The issue may also lie in the frontend as the span time does not always align with actual request time. This is because javascript won't always process an http request as soon as its response is ready due to it being asynchronous. There may be some other tasks that delay the time it takes for javascript to get to.
```

Comment:
```suggestion
- The issue may also lie in the frontend, as span time does not always align with actual request time. This is because JavaScript doesn't always process an HTTP request as soon as its response is ready because it's asynchronous and may be processing other tasks.
```